packages/monaco/src/browser/monaco-quick-input-service.ts

```diff
@@ -316,6 +317,17 @@ export class MonacoQuickInputService implements QuickInputService {
         const monacoPicks: Promise<QuickPickInput<IQuickPickItem>[]> = new Promise(async resolve => {
             const updatedPicks = (await picks).map(pick => {
                 if (pick.type !== 'separator') {
+                    const icon = pick.iconPath;
+                    // TODO: Other kind of icons (URI and URI dark/light) shall be supported once monaco editor has been upgraded.
+                    // see https://github.com/eclipse-theia/theia/pull/12945#issue-1913645228
```

Comment:
Suggestion: We have a dedicated tag for this, see [`@monaco-uplift`](https://github.com/eclipse-theia/theia/blob/ba3722b04ff91eb6a4af6a571c9e263c77cdd8b5/packages/plugin-ext/src/main/browser/theme-icon-override.ts#L19-L20). Please use that one.

Reply:
Thanks @msujew! I pushed a new version with @monaco-uplift tag.